.golangci.yml

```diff
@@ -0,0 +1,44 @@
+run:
```

Comment:
How do we run the linter locally? Shouldn't it run as part of `make lint`?

Reply:
`golangci-lint run` - https://golangci-lint.run/usage/install/#local-installation

I didn't want to change `make lint`, because that seems opinionated and is used by CI to fail the PR.

There are some discussions about linting and if it should fail CI. You'll always have to exclude edge cases, especially when you go beyond linking into static analysis. Which is very useful for coding, but you might not want to fail tests just because cyclomatic complexity checks don't understand ginkgo contexts...
On the other hand, I'm going to fix the linter errors (spellcheck, errchck) anyhow, why not fail the run then.

Didn't want to decide that by myself. Maybe we merge this and look at the results from golangci-lint on Github and decide which ones should fail PRs together?